### PR TITLE
all: some simplification

### PR DESCRIPTION
**Pull request by ferhatelmas**
* don't check err, just return
  * don't check bool to bool, check bool directly

### CODE REVIEW

File: server/cmd_selected.go [hunk 1]
```diff
@@ -52,12 +52,7 @@ func (cmd *Close) Handle(conn Conn) error {
 	ctx.Mailbox = nil
 	ctx.MailboxReadOnly = false
 
-	if err := mailbox.Expunge(); err != nil {
-		return err
-	}
-
-	// No need to send expunge updates here, since the mailbox is already unselected
```
Comment (emersion) [inline server/cmd_selected.go]: Please keep this comment!

**Comment by ferhatelmas** [comment]
@emersion Added it back.

**Comment by emersion** [comment]
Thanks for your contribution! 👍

**Comment by ferhatelmas** [comment]
My pleasure :heart:

**Pull request merged**